brightlayer-ui/brightlayer-ui-cli
# BLUI-3261 Upgrade to react native v0.70.0

Fixes BLUI-3261.


#### Changes proposed in this Pull Request:
- Update to react-native@0.70.0


#### Screenshots:


#### To Test:
- `git clone git@github.com:brightlayer-ui/brightlayer-ui-cli.git -b feature/3261-upgrade-to-rn-70`
- `cd brightlayer-ui-cli`
- `yarn && yarn build && yarn link`
- open a new terminal with a directory to generate new projects in
- Test all 6 templates by generating new projects using the following commands (use defaults for unspecified options):
  - `blui new --framework=react-native --name=TestBlankRN70 --template=blank --alpha --language=javascript`
  - `blui new --framework=react-native --name=TestBlankTSRN70 --template=blank --alpha --language=typescript`
  - `blui new --framework=react-native --name=TestRoutingRN70 --template=routing --alpha --language=javascript`
  - `blui new --framework=react-native --name=TestRoutingTSRN70 --template=routing --alpha --language=typescript`
  - `blui new --framework=react-native --name=TestAuthRN70 --template=authentication --alpha --language=javascript`
  - `blui new --framework=react-native --name=TestAuthTSRN70 --template=authentication --alpha --language=typescript`
- Follow the CLI instructions to get the projects running and go through these projects on android and iOS to ensure they all build and function as expected
